packages/help-center/src/components/help-center-chat.tsx

```diff
@@ -40,9 +44,9 @@ export function HelpCenterChat( {
 		<OdieAssistantProvider
 			currentUser={ currentUser }
 			canConnectToZendesk={ canConnectToZendesk }
-			selectedSiteId={ site?.ID as number }
-			selectedSiteURL={ site?.URL as string }
-			selectedConversationId={ conversationId }
+			selectedSiteId={ Number( siteId ) || ( site?.ID as number ) }
+			selectedSiteURL={ siteUrl || ( site?.URL as string ) }
```

Comment:
Just noting here that this happened to me

<img width="297" alt="image" src="https://github.com/user-attachments/assets/ff813a44-5e6e-4a44-9b06-93cff0686cd0" />